tests/scripts/t3000/run_t3000_demo_tests.sh

```diff
@@ -75,6 +75,36 @@ run_t3000_llama3_tests() {
   fi
 }
 
+run_t3000_llama3_vision_tests() {
+  # Record the start time
+  fail=0
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_llama3_vision_tests"
+
+  wh_arch_yaml=wormhole_b0_80_arch_eth_dispatch.yaml
+  # Llama3.2-11B
+  llama11b=/mnt/MLPerf/tt_dnn-models/llama/Llama3.2-11B-Vision-Instruct/
+  n300=N300
+  t3k=T3K
+
+  # Install Vision-specific packages
+  pip install -r models/demos/llama3/reference/llama_models/requirements.txt
```

Comment:
It is possible that because we're using cached python dependencies, the llama submodule dependencies may actually make their way into the dependencies cache on GitHub.

So, if a T3K runner that:
- will run these llama tests
- were to be the next runner on main that runs changes to any of the paths for the cache key (ie. `tt_metal/python_env/requirements-dev.txt`, `pyproject.toml` etc)

then these llama submodule pip dependencies will make it into the cache.

Now, this should be an exceedingly rare event, as single-card pipelines are generally run before T3K ones when developers are working on CI. So, I'm okay with this, but I'm just pointing out that technically bloat / potential conflicts could make it into the cache.

The fix in that case is to just blow away the cache entries and re-run pipelines. So, not a complicated situation to get ourselves out of, anyway.

cc: @ttmchiou @TT-billteng